src/install.rst

```diff
@@ -45,7 +45,7 @@ Install Nextstrain CLI
 
          curl -fsSL --proto '=https' https://nextstrain.org/cli/installer/mac | bash
 
-      You can launch a Terminal by clicking the Launchpad icon in the Dock, typing ``terminal`` in the search field, and clicking Terminal.
+      You can launch a Terminal by searching for ``terminal`` in Spotlight and opening **Terminal.app**.
```

Comment:
_non-blocking_

Do most Mac users know what Spotlight is by name? Or how to invoke/open it? I know two ways and I'm sure you do too, but I wonder if this common knowledge for "normal" users or if we should give an example.

Reply:
Added clarity in [force-push](https://github.com/nextstrain/docs.nextstrain.org/compare/cdc6374530ce66f19ad3f1b24c58d67703a21145..29f44b96d8aa1a18df8409314fd934edcda8137e).